static/app/components/actions/archive.tsx

```diff
@@ -136,7 +142,19 @@ function ArchiveActions({
             disabled={disabled}
           />
         )}
-        menuTitle={t('Archive')}
+        menuTitle={
+          <MenuWrapper>
+            {t('Archive')}
+            <StyledExternalLink
+              href="https://sentry-docs-git-update-beta-test-archiving.sentry.dev/product/issues/states-triage/escalating-issues/"
+              onClick={event => {
+                event.stopPropagation();
```

Comment:
What's this needed for?

Reply:
whoops thanks i was playing with something else